scripts/machine-launch-script.sh

```diff
@@ -74,12 +74,21 @@ sudo python3 -m pip install moto==2.2.17
 sudo python3 -m pip install sure==2.0.0
 # needed for the awstools cmdline parsing
 sudo python3 -m pip install pyyaml==5.4.1
+sudo python3 -m pip install mypy==0.931
+sudo python3 -m pip install types-PyYAML==6.0.4
+sudo python3 -m pip install boto3-stubs==1.21.6
+sudo python3 -m pip install botocore-stubs==1.24.6
```

Comment:
I see where the typing stuff is being used all over the place but I don't see a usage of these new stubbing packages.  Did you mean to include these?

Reply:
These stubbing packages were needed to mypy to not throw warnings on `awstools.py`. I'll double check this to make sure they are necessary for this PR (or if we can just push this to when we update `awstools` with the type checking work).

Reply:
Just confirmed this. Without those installed `mypy` errors saying `Skipping analyzing "boto3": module is installed, but missing library stubs or py.typed marker`.